hack/ci-e2e.sh

```diff
@@ -18,12 +18,37 @@ REPO_ROOT=$(realpath "$(dirname "${BASH_SOURCE[0]}")/..")
 
 cd "${REPO_ROOT}" || exit 1
 
-# BMO_E2E_EMULATOR can be set to either "vbmc" or "sushy-tools"
-BMO_E2E_EMULATOR=${BMO_E2E_EMULATOR:-"sushy-tools"}
-# We can choose to use redfish-virtualmedia or redfish as the protocol when using sushy-tools
-SUSHY_TOOLS_PROTOCOL=${SUSHY_TOOLS_PROTOCOL:-"redfish-virtualmedia"}
+# CI originally specified BMO_E2E_EMULATOR only. To avoid breaking it,
+# we need to set BMC_PROTOCOL based on this variable.
+# TODO(lentzi90): Change this to just set the BMO_E2E_EMULATOR based on the
+# BMC_PROTOCOL once CI has been adapted to use it.
+if [[ -z "${BMC_PROTOCOL:-}" ]]; then
+  # If no protocol is specified, we set it based on the emulator.
+  BMO_E2E_EMULATOR=${BMO_E2E_EMULATOR:-"sushy-tools"}
+  if [[ "${BMO_E2E_EMULATOR}" == "sushy-tools" ]]; then
+    BMC_PROTOCOL=${BMC_PROTOCOL:-${SUSHY_TOOLS_PROTOCOL:-"redfish-virtualmedia"}}
+  elif [[ "${BMO_E2E_EMULATOR}" == "vbmc" ]]; then
+    BMC_PROTOCOL=${BMC_PROTOCOL:-"ipmi"}
+  else
+    echo "Invalid e2e emulator specified: ${BMO_E2E_EMULATOR}"
```

Comment:
```suggestion
    echo "FATAL: Invalid e2e emulator specified: ${BMO_E2E_EMULATOR}"
```
Add fatal or error, since were terminating